app/uk/gov/hmrc/apiplatform/modules/uplift/services/UpliftLogic.scala

```diff
@@ -102,4 +101,36 @@ object UpliftLogic {
         }
     }
   }
+
```

Comment:
I wonder if these should not go in the domain lib as they have applicability elsewhere (and we do have some filtering in APM that could benefit)